examples/03_evaluate/als_movielens_diversity_metrics.ipynb

```diff
@@ -153,7 +153,7 @@
     "import pyspark\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.            StructField("Timestamp", StringType()),
</pre><p>Why do we need the type change here? Is there a reason to change the signature of load_spark_df() ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1538/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Oops, this is some artifacts of me testing whether the data types has any impact to the runtime. I can revert the change.